scripts/test_jina_extraction.py

```
@@ -0,0 +1,126 @@
+        print(f"  {url}")
+        print("-" * 80)
+
+        # URL 유효성 검증
+        if not url:
+            print("[오류] URL이 제공되지 않았습니다.")
+            return
+
+        # DB 중복 체크 (옵션)
+        if check_duplicate:
+            if News.objects.filter(url=url).exists():
+                print(f"\n[알림] 이미 저장된 기사입니다 (DB 중복).")
```

Ruff (0.14.11)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)